.bcr/presubmit.yml

```diff
@@ -0,0 +1,23 @@
+bcr_test_module:
+  module_path: "."
+  matrix:
+    platform: ["debian10", "ubuntu2004"]
+  tasks:
+    verify_targets:
+      name: Verify Build Targets
+      platform: ${{ platform }}
+      build_targets:
+        - '@maliput//:common'
+        - '@maliput//:math'
+        - '@maliput//:base'
+        - '@maliput//:geometry_base'
+        - '@maliput//:api'
+        - '@maliput//:plugin'
+        - '@maliput//:routing'
+        - '@maliput//:utility'
```

Comment:
are we missing here: `drake` and `test_utilities`?

Reply:
Yes, we are... I made the mistake of copying over targets from the 1.1.1 release. 

Updated.